app/views/assignments/_current.haml

```diff
@@ -1,11 +1,13 @@
 #current-assignment.text-center
-  - if @current_assignment.present?
-    .card.text-white.bg-success
-      %h5.card-title Currently on call: #{@current_assignment.user.last_name}
-  - else
-    .card.text-white.bg-warning
-      %h5.card-title No one is currently on call!
-      - if @fallback_user.present?
-        .card-body #{@fallback_user.last_name} is the fallback.
-  .card
-    .card-body On-call switches at: #{switchover_time(@switchover_hour)}
+  .card.text-white.mb-3
+    - if @current_assignment.present?
+      .card-body.bg-success
+        %h5.card-title Currently on call: #{@current_assignment.user.last_name}
+    - else
+      .card-body.bg-warning
+        %h5.card-title No one is currently on call!
+        - if @fallback_user.present?
+          .card-text #{@fallback_user.last_name} is the fallback.
+  .card.mb-3
+    .card-body
+      .card-text On-call switches at: #{switchover_time(@switchover_hour)}
```

Comment:
Sidenote (no change being requested) these are both conceptually and styled pretty similarly to `alert`s.

Reply:
I've got a PR ready based on this one